Sming/Core/Network/Http/HttpServerConnection.cpp

```diff
@@ -33,6 +33,7 @@ int HttpServerConnection::onMessageBegin(http_parser* parser)
 	// and temp data...
 	reset();
 	bodyParser = nullptr;
+	bodyContentError = false;
```

Comment:
Rename `bodyContentError` to `hasContentError`.

Reply:
done